myapp/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
+import datetime
+import django.utils.timezone 
 
-# Create your models here.
+class Author(models.Model):
+    name = models.CharField(max_length = 30)
+    brithdate = models.DateField()
+    about_author = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+class Blog(models.Model):
+    author = models.ForeignKey(Author, on_delete = models.CASCADE) 
```

Comment:
Please add the `related_name` field. Remember, you have to add `related_name` fields in each foreignkey you created or you will create in the future.